ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/RepresentativeMapper.java

```diff
@@ -38,16 +34,16 @@ public CCDRepresentative to(Representative representative) {
 
     @Override
     public Representative from(CCDRepresentative representative) {
-        Optional<CCDContactDetails> organisationContactDetailsOptional =
-            representative.getOrganisationContactDetails();
-        ContactDetails organisationContactDetails =
-            organisationContactDetailsOptional
-                .isPresent() ? contactDetailsMapper.from(organisationContactDetailsOptional.get()) : null;
+        if (representative == null) {
+            return null;
+        }
 
         return new Representative(
             representative.getOrganisationName(),
             addressMapper.from(representative.getOrganisationAddress()),
-            organisationContactDetails
+            representative.getOrganisationContactDetails().isPresent()
+                ? contactDetailsMapper.from(representative.getOrganisationContactDetails().get())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)